conf/redpanda.yaml

```diff
@@ -46,6 +46,23 @@ redpanda:
   # not recomended for production use
   developer_mode: true
 
+  enable_proxy: true
+
+pandaproxy:
+  # Pandaproxy transport
+  pandaproxy_api:
+    address: "0.0.0.0"
+    port: 8082
+
+  admin_api_doc_dir: ../../../pandaproxy/admin-api-doc
+  api_doc_dir: ../../../src/v/pandaproxy/api/api-doc
+
+pandaproxy_client:
```

Comment:
should this be just `self` or some `optional` type if nullopt == self, else it acts in standalone mode.

Reply:
I kind of like this idea.

Reply:
See latest [force-push](https://github.com/vectorizedio/redpanda/compare/999e14bb5a48e558a2613c2864d9d6b8916987cc..28713931fc8f230eba6146bcfc89f5cb3ac8109c)